src/analysis/retail/priest/shadow/modules/talents/IdolOfCthun.tsx

```diff
@@ -22,17 +25,27 @@ class IdolOfCthun extends Analyzer {
       Events.damage.by(SELECTED_PLAYER_PET).spell(SPELLS.IDOL_OF_CTHUN_MIND_SEAR_DAMAGE),
       this.onDamage,
     );
+    this.addEventListener(
+      Events.resourcechange.by(SELECTED_PLAYER).spell(SPELLS.IDOL_OF_CTHUN_RESOURCE),
+      this.onResource,
+    );
   }
 
   onDamage(event: DamageEvent) {
     this.damage += event.amount + (event.absorbed || 0);
   }
 
+  onResource(event: ResourceChangeEvent) {
+    this.insanity += event.resourceChange;
+  }
+
   statistic() {
     return (
       <Statistic category={STATISTIC_CATEGORY.TALENTS} size="flexible">
         <BoringSpellValueText spellId={TALENTS.IDOL_OF_CTHUN_TALENT.id}>
           <ItemDamageDone amount={this.damage} />
+          <br />
```

Comment:
please wrap in `<div />` if you want a piece of content to occupy its own line instead of inserting brs, which is typically more brittle.

(i don't always catch these in review, and it may never cause problems in this case, but it also makes things a bit easier in the long run if explicit breaks are avoided)

Reply:
I've updated all shadow's statistics to use <div> instead of <br />.
Should I switch to using <div> in other places as well?

Reply:
it is just that generally using explicit `<br/>`s can introduce unintended behavior when dealing with different screen sizes. telling the browser what content should be part of a single block (aka using a `<div />` or another element with `display: block;`) and then letting it handle line breaks typically works better across screen sizes.